src/cpu/reorder/cpu_reorder_regular_u8.cpp

```diff
@@ -22,35 +22,38 @@ namespace cpu {
 
 // clang-format off
 
-const impl_list_map_t regular_u8_impl_list_map REG_REORDER_P({
-    // u8 ->
-    {{u8, data_type::undef, 0}, {
-        REG_FAST_DIRECT_COPY(u8, f32)
-        REG_FAST_DIRECT_COPY(u8, s32)
-        REG_FAST_DIRECT_COPY(u8, bf16)
-        REG_FAST_DIRECT_COPY(u8, s8)
-        REG_FAST_DIRECT_COPY(u8, u8)
-
-        DNNL_X64_ONLY(CPU_REORDER_INSTANCE(x64::jit_blk_reorder_t))
-        DNNL_X64_ONLY(CPU_REORDER_INSTANCE(x64::jit_uni_reorder_t))
-
-        DNNL_AARCH64_ONLY(CPU_REORDER_INSTANCE(aarch64::jit_uni_reorder_t))
-
-        DNNL_NON_X64_ONLY(REG_SR_BIDIR(u8, any, f32, nChw16c))
-        DNNL_NON_X64_ONLY(REG_SR_BIDIR(u8, any, s32, nChw16c))
-        DNNL_NON_X64_ONLY(REG_SR_BIDIR(u8, any, bf16, nChw16c))
-        DNNL_NON_X64_ONLY(REG_SR_BIDIR(u8, any, s8, nChw16c))
-        DNNL_NON_X64_ONLY(REG_SR_BIDIR(u8, any, u8, nChw16c))
-
-        REG_SR(u8, any, f32, any, fmt_order::any, spec::reference)
-        REG_SR(u8, any, s32, any, fmt_order::any, spec::reference)
-        REG_SR(u8, any, bf16, any, fmt_order::any, spec::reference)
-        REG_SR(u8, any, u8, any, fmt_order::any, spec::reference)
-        REG_SR(u8, any, s8, any, fmt_order::any, spec::reference)
-
-        nullptr,
-    }},
-});
+const impl_list_map_t &regular_u8_impl_list_map() {
+    static const impl_list_map_t the_map = REG_REORDER_P({
+        // u8 ->
+        {{u8, data_type::undef, 0}, {
+            REG_FAST_DIRECT_COPY(u8, f32)
+            REG_FAST_DIRECT_COPY(u8, s32)
+            REG_FAST_DIRECT_COPY(u8, bf16)
+            REG_FAST_DIRECT_COPY(u8, s8)
+            REG_FAST_DIRECT_COPY(u8, u8)
+
+            DNNL_X64_ONLY(CPU_REORDER_INSTANCE(x64::jit_blk_reorder_t))
+            DNNL_X64_ONLY(CPU_REORDER_INSTANCE(x64::jit_uni_reorder_t))
+
+            DNNL_AARCH64_ONLY(CPU_REORDER_INSTANCE(aarch64::jit_uni_reorder_t))
+
+            NON_X64_ONLY(REG_SR_BIDIR(u8, any, f32, nChw16c))
```

Comment:
`DNNL_` prefix got lost here, can you fix it?

Reply:
Sure. Thanks for noticing. The regexp must have been faulty.